### DISCUSSION TITLE
Throwable is never a GraphQLError

### DISCUSSION THREAD

**Discussion by eduarddrenth** [graphql-java-kickstart/graphql-java-servlet#409] [Q&A] [answered]
If I am not mistaken a throwable will never be a GraphQLError...

https://github.com/graphql-java-kickstart/graphql-java-servlet/blob/cce4709859c0ab9064e6a087a00614d357776aba/graphql-java-kickstart/src/main/java/graphql/kickstart/execution/error/DefaultGraphQLErrorHandler.java#L29

**Comment by oliemansm** [comment, marked as answer]
`GraphQLError` is just an interface and anybody is free to create a custom exception which also implements the `GraphQLError` interface. So think that's the main reason this condition exists.

I'll convert this issue to a Discussion btw.

**Reply by eduarddrenth** [reply]
fair enough, I see graphql-java has a lot of Throwables implementing GraphqlError
